packages/components/atoms/f-card/stories/card.stories.js

```diff
@@ -1,41 +1,40 @@
-import {
-    withKnobs, boolean, select, text
-} from '@storybook/addon-knobs';
-import Card from '../src/components/Card.vue';
 import { withA11y } from '@storybook/addon-a11y';
+import Card from '../src/components/Card.vue';
 
 export default {
     title: 'Components/Atoms',
-    decorators: [withKnobs, withA11y]
+    decorators: [withA11y],
+    parameters: {
+        backgrounds: {
+            values: [
+                { name: 'Grey', value: '#f5f5f5' },
+                { name: 'Dark', value: '#333' }
+            ],
+            default: 'Grey'
+        },
+        notes: 'some documentation here'
```

Comment:
Out of interest, why do we have `notes: 'some documentation here'` in some stories?

Reply:
It was in there before, but agree it shouldn't be. Will remove it now